src/sql/common/constants.ts

```diff
@@ -14,4 +14,5 @@ export const RestoreFeatureName = 'restore';
 export const BackupFeatureName = 'backup';
 
 export const MssqlProviderId = 'MSSQL';
+export const notebookModeId = 'jupyter-notebook';
```

Comment:
As discussed offline, let's call this 'notebook', and fix the package.json in Notebooks extension to use that too (and only have the alias 'Notebook' instead of Jupyter Notebook and IPython Notebook.